app/assets/javascripts/angular/controllers/dashboard_ctrl.js

```diff
@@ -30,6 +30,31 @@ angular.module("Prometheus.controllers")
     $scope.$broadcast('redrawGraphs');
   }
 
+  $scope.generateStaticDashboard = function(event) {
+    // TODO: When the input with the static dashboard link is visible,
+    // should clicking the "generate" button close the input, or
+    // generate a NEW static link?
+    $scope.generatingPermalink = true;
+    $http.post(window.location.origin + "/static", {
```

Comment:
I was confused by the terminology "static" vs. "dynamic" and assumed it meant the other way around: dashboards generated "dynamically" by clicking the permalink button vs. "static" ones which are the "real" ones :)

How about e.g. "/perm" for the URLs and "permalink" for the boolean DB column? Now just needs a good name for the "normal" dashboards :)